templates/forms/DropdownField.ss

```diff
@@ -1,5 +1,5 @@
 <select $AttributesHTML>
 <% loop $Options %>
-	<option value="$Value.XML"<% if $Selected %> selected="selected"<% end_if %><% if $Disabled %> disabled="disabled"<% end_if %>><% if Title %>$Title.XML<% else %>&nbsp;<% end_if %></option>
```

Comment:
It looks as thought this broke the test at https://travis-ci.org/silverstripe/silverstripe-framework/jobs/67920076